forms-flow-api/src/formsflow_api/models/employee_data.py

```diff
@@ -5,7 +5,7 @@ class EmployeeData():
   # data: Response retrieved from the ODS
   # Spec of possible values can be found at: https://analytics-testapi.psa.gov.bc.ca/apiserver/api.rst#Datamart_Telework_employee_demo
   def __init__(self, data):
-   self.displayName = data["first_name"] + " " + data["last_name"]
+   self.displayName = " ".join(filter(None, (data["first_name"], data["last_name"])))
```

Comment:
This would prevent the whole endpoint to break if either first_name or last_name was null.